hudi-common/src/main/java/org/apache/hudi/common/bloom/HashFunction.java

```diff
@@ -16,18 +16,51 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+/**
+ * Copyright (c) 2005, European Commission project OneLab under contract 034819
+ * (http://www.one-lab.org)
+ * <p>
+ * All rights reserved.
```

Comment:
Is this needed in the license header? Was it present in the original classes?

Reply:
Yes, this is present in the original classes.  To maintain the license, I copied that for all relevant classes.